lib/experimental/register-webfonts-from-theme-json.php

```diff
@@ -152,7 +152,8 @@ function gutenberg_add_registered_webfonts_to_theme_json( $data ) {
 
 	foreach ( $to_add as $slug ) {
 		$font_faces_for_family = $font_families_registered[ $slug ];
-		$family_name           = $font_faces_for_family[0]['font-family'];
+		$font_family           = $font_faces_for_family[0]['font-family'];
+		$name           = ! empty( $font_faces_for_family[0]['name'] ) ? $font_faces_for_family[0]['name'] : $font_faces_for_family[0]['font-family'];
```

Comment:
Alignment is wrong.

Reply:
thanks, [fixed](https://github.com/WordPress/gutenberg/pull/46404/commits/17983a554d65dbe0fb1586c23555a1eac52ee285)